Sources/PackageFingerprint/FilePackageFingerprintStorage.swift

```diff
@@ -398,11 +398,24 @@ extension PackageReference: FingerprintReference {
         }
 
         let canonicalLocation = CanonicalPackageLocation(sourceControlURL.absoluteString)
-        let locationHash = String(format: "%02x", canonicalLocation.description.hashValue)
+        // Cannot use hashValue because it is not consistent across executions
+        let locationHash = String(format: "%02x", canonicalLocation.description.stableHash)
```

Comment:
could we use the same hash function on string as we do for repository storage location?

Reply:
can you point me to that code please?

Reply:
https://github.com/apple/swift-package-manager/pull/6780/files#diff-989b9f25e2078b5e7709d63f4d57027c8263c49ac74e2200413ee6f6dc878db5R571

Reply:
thanks! amended https://github.com/apple/swift-package-manager/pull/6784/commits/aac13e7f8905c88a7b2e670a824eb1762a996c7e